cabot/cabotapp/models.py

```diff
@@ -212,7 +220,7 @@ def alert(self):
                     if self.last_alert_sent and (timezone.now() - timedelta(minutes=settings.NOTIFICATION_INTERVAL)) \
                             < self.last_alert_sent:
                         return
-                elif self.overall_status in (self.CRITICAL_STATUS, self.ERROR_STATUS):
+                elif self.overall_status in (self.CRITICAL_STATUS, self.ERROR_STATUS, self.ACKED_STATUS):
```

Comment:
It certainly could...I don't think it matters much, since this is only for repeat alerts on acked services, which I think every alert plugin will ignore.